aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -384,6 +384,9 @@ webBuilder.ConfigureKestrel(serverOptions =>
 });
 ```
 
+> [!NOTE]
+> Endpoints created by calling <xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.Listen*> or <xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.ListenUnixSocket*> **before** calling <xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.ConfigureEndpointDefaults*> will not be updated with the new defaults.
```

Comment:
Why call out ListenUnixSocket separately? `Listen*` should be adequate.

Reply:
```suggestion
> Endpoints created by calling <xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.Listen*> **before** calling <xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.ConfigureEndpointDefaults*> will not have the defaults applied.
```

Reply:
I guess because other places have cited them separately. I'm OK with dropping it though. Makes it a little cleaner.